src/documents/customPreview.md

```diff
@@ -0,0 +1,951 @@
+    });
+</script>
+```
+
+## Route Setup
+
+Its nice to see what our preview will look like in the web browser, but now we need to get it into the preview of the First Discovery Tool.  To do this we will need to go into the firstDiscoveryEditor.js file and find the gpii.firstDiscovery.getPreviewUrl function and add an additional else if statement to handle our new preview content page. 
+
+__STEP 1: Navigate to__ your __firstDiscoveryEditor.js__ file and open it. Make the __following changes__ to the getPreviewUrl function.
+
+__firstDiscoveryEditor.js__
+
+```javascript
+gpii.firstDiscovery.getPreviewUrl = function (that, languageCode) {
+    var baseurl = "../../src/html/";
+    var previewname = gpii.firstDiscovery.getPreviewName();
+    var previewpage = "searchpreview.html"; //default
+    if (previewname === "search") {
+        previewpage = "searchpreview.html";
+    } else if (previewname === "electron") {
+        previewpage = "electronpreview.html";
+    } else if (previewname === "example") {
+        previewpage = "examplePreview.html";
+    }
+
+    //unsupported languages will return english preview.
+    //TODO: create previews for remaining languages, de-DE, nl-NL, sv-SE
+    if(languageCode == "de-DE"){
+        languageCode = "en-US";
+    }
+    if(languageCode == "nl-NL"){
+        languageCode = "en-US";
+    }
+    if(languageCode == "sv-SE"){
+        languageCode = "en-US";
+    }
+
+    var previewQueryString = "?lang=" + languageCode;
+
+    return encodeURI(baseurl + previewpage + previewQueryString);
+};
+```
+
+__STEP 2: Start up your first-discovery-server__ by navigating to your local repository and __run :
+node index.js__
+
+![Command Prompt demonstrating successful startup of local first discovery server](images/customPreview3.png)
+
+__STEP 3:__ Navigate to your __Chrome browser__ and navigate to http://localhost:8088/demos/prefsServerIntegration/index.html?preview=example
+
+![The first Discovery Tool with the new custom preview html that we created earlier](images/customPreview4.png)
+
+As you can see, we now have a route to our First-Discovery-Tool Preview. The First Discovery Tool allows the users to change to a specific preview content page by editing the end of the url to any of those specified in the getPreviewUrl function that we changed. This includes __electron__, __search__ and now __example__.
+
+SEARCH: http://localhost:8088/demos/prefsServerIntegration/index.html?preview=search
+
+ELECTRON: http://localhost:8088/demos/prefsServerIntegration/index.html?preview=electron
+
+EXAMPLE: http://localhost:8088/demos/prefsServerIntegration/index.html?preview=example
+
+## Extracting and Storing the Page Content
+
+The First Discovery Tool supports multiple languages, so the next step is to extract all of the words that could be translated into a separate message file (JSON file) that is easily translated and maintained, as opposed to several whole html pages in different languages.
```

Comment:
Somewhere, this looks like a good spot, we should explicitly state that the language is provided to the preview via the lang querystring variable and is of the form en-US/es-MX